
<launch>
    <node name="simulated_odometry" pkg="my_robot_navigation" type="simulated_odometry.py" output="screen"/>
</launch>